### PR TITLE
TT-65 feat(TT-65): added follow-up node and example

### PR DESCRIPTION
**Pull request by dmiller15**
r? @ProfOak

### CODE REVIEW

File: gdcdictionary/schemas/_terms.yaml [hunk 1]
```diff
@@ -352,6 +421,17 @@ cause_of_death:
     cde_version: 3.0
     term_url: "https://cdebrowser.nci.nih.gov/CDEBrowser/search?elementDetails=9&FirstTimer=0&PageId=ElementDetailsGroup&publicId=2554674&version=3.0"
 
+cause_of_response:
+  description: >
+    A specific regimen or line of therapy responsible for the patient's observed improvement achived
```
Comment (ProfOak) [inline gdcdictionary/schemas/_terms.yaml]: achieved

File: gdcdictionary/schemas/_terms.yaml [hunk 2]
```diff
@@ -786,6 +973,18 @@ height:
     cde_version: 4.1
     term_url: "https://cdebrowser.nci.nih.gov/CDEBrowser/search?elementDetails=9&FirstTimer=0&PageId=ElementDetailsGroup&publicId=649&version=4.1"
 
+hemoglobin:
+  description: >
+    The value (g/dL) for a the medical procedure that involves testing a sample of blood for
```
Comment (ProfOak) [inline gdcdictionary/schemas/_terms.yaml]: a the

File: gdcdictionary/schemas/_terms.yaml [hunk 3]
```diff
@@ -786,6 +973,18 @@ height:
...
+    hemoglobun, red respiratory protein of erythrocytes, to help determine a diagnosis, plan
```
Comment (ProfOak) [inline gdcdictionary/schemas/_terms.yaml]: hemoglobun

File: gdcdictionary/schemas/_terms.yaml (hunk 1 above)
Comment (ProfOak) [inline gdcdictionary/schemas/_terms.yaml]: achived

Comment (dmiller15) [inline gdcdictionary/schemas/_terms.yaml]: 👍

Comment (dmiller15) [inline gdcdictionary/schemas/_terms.yaml]: 👍

File: gdcdictionary/schemas/_terms.yaml (hunk 2 above)
Comment (dmiller15) [inline gdcdictionary/schemas/_terms.yaml]: 👍

File: gdcdictionary/schemas/_terms.yaml (hunk 3 above)
Comment (dmiller15) [inline gdcdictionary/schemas/_terms.yaml]: 👍

**Pull request merged**